src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -82,38 +83,47 @@ class RoutedSavableReviewPage extends React.Component {
       return saveLink;
     }
 
-    let InlineErrorComponent;
-    if (typeof errorText === 'function') {
-      InlineErrorComponent = errorText;
-    } else if (typeof errorText === 'string') {
-      InlineErrorComponent = () => <p>{errorText}</p>;
-    } else {
-      InlineErrorComponent = () => (
-        <p>
-          If it still doesn’t work, please <CallHRC />
-        </p>
-      );
-    }
-
-    return (
-      <div className="usa-alert usa-alert-error schemaform-failure-alert">
-        <div className="usa-alert-body">
-          <p className="schemaform-warning-header">
-            <strong>We’re sorry. We can't submit your form right now.</strong>
-          </p>
+    const DefaultErrorMessage = () => {
+      let InlineErrorComponent;
+      if (typeof errorText === 'function') {
+        InlineErrorComponent = errorText;
+      } else if (typeof errorText === 'string') {
+        InlineErrorComponent = () => <p>{errorText}</p>;
+      } else {
+        InlineErrorComponent = () => (
           <p>
-            We’re working to fix the problem. Please make sure you’re connected
-            to the Internet, and then try saving your form again. {saveLink}.
+            If it still doesn’t work, please <CallHRC />
           </p>
-          {!user.login.currentlyLoggedIn && (
+        );
+      }
+
+      return (
+        <div className="usa-alert usa-alert-error schemaform-failure-alert">
+          <div className="usa-alert-body">
+            <p className="schemaform-warning-header">
+              <strong>We’re sorry. We can't submit your form right now.</strong>
+            </p>
             <p>
-              If you don’t have an account, you’ll have to start over. Try
-              submitting your form again tomorrow.
+              We’re working to fix the problem. Please make sure you’re
+              connected to the Internet, and then try saving your form again.{' '}
+              {saveLink}.
             </p>
-          )}
-          <InlineErrorComponent />
+            {!user.login.currentlyLoggedIn && (
+              <p>
+                If you don’t have an account, you’ll have to start over. Try
+                submitting your form again tomorrow.
+              </p>
+            )}
+            <InlineErrorComponent />
+          </div>
         </div>
-      </div>
+      );
+    };
+
+    return CustomSubmissionError ? (
+      <CustomSubmissionError location={location} form={form} user={user} />
+    ) : (
+      <DefaultErrorMessage />
```

Comment:
Shouldn't we add a test for this?

I'd also be interested in seeing an example of adding a CustomSubmissionError in the form config.

Reply:
We cannot test this due to not being able to do a full mount in this component to see the children due to pre-existing tech debt as stated in the above comment.

As for examples, I have another PR with a custom error component implemented as well as updated documentation with an example.

Implementation PR: 
https://github.com/department-of-veterans-affairs/vets-website/pull/13298

Documentation PR:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/307

Reply:
OK thanks @LinnJS .